universal-login-sdk/lib/sdk.js

```diff
@@ -34,7 +35,7 @@ class EthereumIdentitySDK {
     throw new Error(`${responseJson.error}`);
   }
 
-  async addKey(to, publicKey, privateKey, transactionDetails, keyPurpose = MANAGEMENT_KEY) {
+  async addKey(to, publicKey, privateKeys, transactionDetails, keyPurpose = MANAGEMENT_KEY) {
```

Comment:
This is not a good idea. That means that one client/device has access to all need keys, which contradicts the purpose of multisig. 

The way to implement it on SDK level is:
- one device requests an execution
- other devices get a notification
- other devices can confirm (by signing the same data and sending the signature to relayer)
- relayer gathers signatures and triggers execution when complete

Let's revert those changes now and go back to them in the next PR. 
We can do a call to discuss it.

I will use tag #mutipleKeys, to mark related changes.